package.json

```diff
@@ -48,6 +49,7 @@
   "homepage": "https://github.com/mondaycom/monday-ui-style#readme",
   "devDependencies": {
     "@types/jest": "^27.4.1",
+    "changelog": "^1.4.2",
```

Comment:
I know it's just for a changelog, but is there an alternative for it? or is it something we are using in more places?
it isn't maintained for 5 years and has ±1100 weekly downloads

Reply:
I've tried some others packages, but most of them are relying on git tags and we just don't have them there... This one is relying on github/npm api instead